### ISSUE TITLE
Dev Branch - Cannot Enter and Save Carbs When There Isn't an Insulin Pump Added and Configured

### ISSUE THREAD

**Issue by rolandet** [LoopKit/Loop#1633]
**Describe the bug**

I was switching from the OmniPod Eros to the OnmiPod DASH using a Dev Branch build from March 2nd and after removing the OmniPod Eros pump from the Configuration settings I was not able to enter Carbs.

**Attach an Issue Report**

**To Reproduce**
1. Without a Pump configured attempt to add Carbs
2. Click "Save..." and it will not let you save the Carb entry. I don't remember the exact error. I'll grab a screen capture the next time if that is needed.

**Expected behavior**
I would expect that you should be able to log your Carbs even if a Pump is not added and configured.

**Screenshots**
I did not capture a screen capture at the time.

**Phone**
 - Hardware: iPhone 12
 - OS Version: iOS 15.3.1

**Loop Version**
 - Version Number: 2.3.0 (57) Built from March 2nd Dev Branch
 - Repo: LoopKit/Loop WorkSpace

**CGM**
 - Device: Dexcom G6
 - Manager app: Dexcom App

**Pump**
 - Manufacturer:  Insulet
 - Model: DASH
 - Firmware version: 3.27.0

**Additional context**

**Comment by rolandet** [comment]
Attaching a screen capture with the message when trying to enter carbs with no pump configured.
![IMG_0437](https://user-images.githubusercontent.com/4328088/160317811-48abe617-5250-4ef8-84dc-8ea30aceef8e.PNG)

**Comment by marionbarker** [comment]
I tested with the latest version of Loop-dev: Workspace commit e68c4e9.

This is still an issue.
I tested with PR #1678 applied and that PR fixes this issue - I will update this use case in that PR.

Use Case for needing to add carbs when no pump is present.
* looper is between pumps for some reason - either no pump is configured, or no pod is paired but wants Loop to continue predicting future BG
